### ISSUE TITLE
Plotting Training Results

### ISSUE THREAD

**Issue by dodo269696** [tobybreckon/fire-detection-cnn#11]
**Describe the issue**
< I have re-trained the FireNet with my own dataset, at the end of last epoch (training), I want to see accuracy- epoch, loss - epoch, validation loss - epoch graphs. Tried history= model.fit() but could not plot any graph by using it>.

**Computing Environment (please complete the following information):**
 - OS: [Windows10 ]

I am using Google Colab for training process

**Comment by tobybreckon** [comment]
This is a general query about the deep learning environment you are using (which is? - tflearn, keras ... ?), not the software provided here - please refer to the appropriate manual / user examples on this.
